src/test/java/jenkins/plugins/openstack/PluginTestRule.java

```diff
@@ -99,7 +100,7 @@ public static SlaveOptions dummySlaveOptions() {
             dummyUserData("dummyUserDataId");
         }
         return new SlaveOptions(
-                new BootSource.VolumeSnapshot("id"), "hw", "nw1,mw2", "dummyUserDataId", 1, "pool", "sg", "az", 1, null, 10,
+                new BootSource.VolumeSnapshot("id"), "hw", "nw1,mw2", "dummyUserDataId", 1, 2, "pool", "sg", "az", 1, null, 10,
```

Comment:
I suspect this might intervene with existing tests that does not expect nodes can be launched for other reasons than explicit provisioning and responding to queue demand. This would be better either left to defaults of not provisioning or having the `JCloudsPreCreationThread` disabled for unittesting[1] unless explicitly invoked.

[1] https://github.com/jenkinsci/node-sharing-plugin/blob/03c9d03f8778ff8de250b2715e57b91fa440c4c7/backend-plugin/src/main/java/com/redhat/jenkins/nodesharingbackend/Pool.java#L218

Reply:
Btw, is it valid to have `instancesMin > instanceCap`? What would happen?

Reply:
Not an invalid config.  In this case `instanceCap` machines (i.e. `min(instancesMin, instanceCap)`) would be pre-provisioned.

Reply:
> I suspect this might intervene with existing tests [...]

My understanding is that this particular instance *must not* collide with the defaults (and 0 is the default for `instancesMin`).

Reply:
I think I will follow your suggestion and simply use `Functions.getIsUnitTest() ? Long.MAX_VALUE : MIN;`.

Reply:
Done.